diagnostic.md

```diff
@@ -51,13 +57,16 @@ Write the `curl` request you'd use to delete a single donut, using whatever
 ID you'd like.
 
 ```sh
-# your answer here
+# curl --include --request DELETE http://www.example.com/donuts
```

Comment:
This has a similar issue with the previous question.  An `id` needs to be present.